src-tauri/src/lib.rs

```diff
@@ -350,7 +350,7 @@ pub fn check_origin_running() -> bool {
 /// Checks if Northstar process is running
 pub fn check_northstar_running() -> bool {
```

Comment:
suggestions doesn't work so here is another one liner :trollface: 
```rust
pub fn check_northstar_running() -> bool {
    sysinfo::System::new_all().processes_by_name("NorthstarLauncher.exe")
}
```

Reply:
Implemented in 4c5fc48c41370ad17aef1bbad2fd7313ee49808a